hermeto/core/package_managers/yarn_classic/resolver.py

```diff
@@ -131,8 +133,15 @@ class FilePackage(_BasePackage, _PathMixin):
     @property
     def purl(self) -> str:
         """Return package URL."""
-        repo_id = get_repo_id(self.path.root)
-        qualifiers = {"vcs_url": repo_id.as_vcs_url_qualifier()}
+        qualifiers = {}
```

Comment:
A nitpick: L136 to L144 seem to be present in all three classes, it would make more sense to pull them to some common base.